crates/forge/tests/integration/test_state.rs

```diff
@@ -362,7 +375,7 @@ fn test_cant_call_test_contract() {
                     }
         
 
-                    #[external(v0)]
+                    #[abi(embed_v0)]
```

Comment:
So it is possible to write abi just above one function 😮 Maybe we shouldn't make traits with just one function then